scripts/bundle-build/components/notifications/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+outputDir=$(basename $PWD)-artifacts
+mkdir -p $outputDir/maven
+cd notifications
+
+./gradlew publishToMavenLocal -PexcludeTests="**/SesChannelIT*" -Dopensearch.version=$1
```

Comment:
Hmm, I'm slightly conerned about repo specific logic being in this common location that could be overlooked when considering testing / validating build configuration.  As this how bundles are built, thoughts on a way to make sure the typical normal build is identical?

Reply:
This is a build script that is specific to the notifications repo.  It should eventually get moved to that repository but it is here now to get this off the ground.

Reply:
We will be moving these into the individual repos at the very least.